test/workos/organizations/organizations_test.exs

```diff
@@ -0,0 +1,121 @@
+defmodule WorkOS.OrganizationsTest do
+  use ExUnit.Case
+  doctest WorkOS.Organizations
+  import Tesla.Mock
+
+  alias WorkOS.Organizations
+
+  describe "#list_organizations/1" do 
+    setup do
+      mock(fn
+        %{method: :get, url: "https://api.workos.com/organizations"} ->
+          %Tesla.Env{status: 200, body: "Success"}
```

Comment:
Would we want to mock out the body in the successful requests here? Like here we have a fixture - is there a similar pattern in Elixir? https://github.com/workos/workos-node/blob/main/src/organizations/organizations.spec.ts#L19

Reply:
Yup, and I think we should implement something like https://github.com/msz/hammox, our mocking structure is error prone atm

Reply:
Sounds good! I think that's fine for a follow up task 👍

Reply:
I think we can continue to use Tesla mock but actually define the fixtures as modules attributes on the test module https://elixir-lang.org/getting-started/module-attributes.html#as-constants

Example:
````ex
@list_organizations_response %{ ... }
````

Reply:
But I actually think that module attributes aren't the best here in case we want to reference this fixture somewhere else, wouldn't make sense to import the test module from another test module? 🤯 

Maybe having a folder with JSON files would make sense! I'm going to add a Linear issue for that

Reply:
Sounds good 👍